core/src/main/java/hudson/model/Computer.java

```diff
@@ -448,7 +448,6 @@ public final void launch() {
     /**
      * CLI command to reconnect this node.
      */
-    @CLIMethod(name="connect-node")
     public void cliConnect(@Option(name="-f",usage="Cancel any currently pending connect operation and retry from scratch") boolean force) throws ExecutionException, InterruptedException {
```

Comment:
I'm not sure if @Deprecated should be here. There are couple of CLI commands extracted from core to CLI already and none of the original code was marked as deprecated as I'm aware.

Moving args4j to javadoc :+1: